data/misc/coco/coco_sample.json

```diff
@@ -0,0 +1,120 @@
+{
```

Comment:
shouldn't we not be saving anything under the `data` folder?  i think `data` is ignored in our gitignore (and if its not, it probably should be)

Reply:
Yeah, was wondering about that also, what to download, and what to put into the repo. So far, do we ever download anything other than images? We of course already have the data/misc folder, but guess all that's in there so far is NOT for testing but for user to run the notebook.

Reply:
An alternative (which I sort of like best of all options) could be that I create a new file ./tests/resources.py and copy the content of the .json file there surrounded by """ quotes, import that in conftest.py, and then just write it to a temporary file during testing

Reply:
i prefer the second thing you suggested. actually what we currently have under `data/misc` (which is only used for action rec) should be downloaded from blob and not saved directly in the repo.

Reply:
in a later PR for action rec, i'll likely delete the files inside `data/misc`

Reply:
I think uploading the `data/*` to a blob makes much sense as I remember seeing a thread on the OS AI recommenders channel that compared the sizes of our repositories and this will help reduce the size of our repo.
`272M    computervision-recipes`
`162M    forecasting`
`50M     nlp-recipes`
`33M     recommenders`

Reply:
Removed the file and instead added string as suggested above.

Reply:
@youngpark yikes, didn't realize ours was so big